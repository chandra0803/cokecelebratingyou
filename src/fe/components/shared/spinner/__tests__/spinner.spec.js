import React from 'react';
import Spinner from '../spinner.js';
import renderer from 'react-test-renderer';

test( 'Spinner renders', () => {
    const component = renderer.create(
        <div>
            <Spinner />
        </div>
    );

    const tree = component.toJSON();
    expect( tree ).toMatchSnapshot();
} );
