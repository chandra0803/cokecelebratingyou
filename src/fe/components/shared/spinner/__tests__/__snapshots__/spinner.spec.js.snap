// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner renders 1`] = `
<div>
  <div
    className="spincover pageLoading"
    id="pageLoadingSpinner"
  >
    <img
      alt="Loading..."
      src="assets/img/pageLoadingSpinner.gif"
    />
  </div>
</div>
`;
